### PR TITLE
chore: app landing page is added

### PR DESCRIPTION
**Pull request by Sushilverma002**
## Related Issue

close issue-> #2013 
## Description

app landing page PROJECT is added
## Screenshots

image

### CODE REVIEW

**Comment by Sushilverma002** [comment]
@priyankarpal have a look!!

File: DB/projects.json [hunk 1]
```diff
@@ -5398,5 +5398,19 @@
...
+    "github_username": "Sushilverma002",  
+    "Social_media": {
+      "LinkedIn": "https://www.linkedin.com/in/sushil-verma-1b228423a/"
+    },
+    "Projects": [
+      {
+        "link": "https://github.com/Sushilverma002/applanding",
+        "title": "App Landing Page",
+        "description": "An App landing page which shows the features of your app on website.",
+        "tech": ["html", "css", "bootstrap","js"]
```
Comment (priyankarpal) [inline DB/projects.json:5412]: change to `javascript`

```suggestion
        "tech": ["html", "css", "bootstrap","javascript"]
```

**Review by priyankarpal — CHANGES_REQUESTED** [review]

**Comment by Sushilverma002** [comment]
@priyankarpal changes a done have a look.

**Review by priyankarpal — APPROVED** [review]

**Comment by Sushilverma002** [comment]
Please add level label and GGSoC'23

**Comment by Sushilverma002** [comment]
@priyankarpal

**Comment by Sushilverma002** [comment]
@priyankarpal please add level label and GGSoC'23 label so that I get points for this Pr

**Comment by priyankarpal** [comment]
> @priyankarpal please add level label and GGSoC'23 label so that I get points for this Pr

We are not providing any labels for project additions as they are not associated with solving any issues.( according to gssoc23 rules)

**Pull request merged**